trunk/SUAVE/Components/Energy/Networks/Battery_Cell_Cycler.py

```diff
@@ -0,0 +1,383 @@
+## @ingroup Components-Energy-Networks
+# Battery_Cell_Cycler.py
+# 
+# Created: Apr 2021, M. Clarke
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# suave imports
+import SUAVE
+
+# package imports
+import numpy as np
+from .Network import Network
+from SUAVE.Components.Energy.Storages.Batteries.Constant_Mass import Lithium_Ion_LiNCA_18650      
+from SUAVE.Components.Energy.Storages.Batteries.Constant_Mass import Lithium_Ion_LiNiMnCoO2_18650   
+from SUAVE.Components.Energy.Storages.Batteries.Constant_Mass import Lithium_Ion_LiFePO4_38120    
+from SUAVE.Core import Data
+
+# ----------------------------------------------------------------------
+#  Network
+# ----------------------------------------------------------------------
+
+## @ingroup Components-Energy-Networks
+class Battery_Cell_Cycler(Network):
+    """ This is a test bench for new batteries 
+    
```

Comment:
update description so we know what it does

Reply:
More descriptive model added